packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -106,18 +119,50 @@ class AttachCommand extends FlutterCommand {
       : null;
 
     Uri observatoryUri;
+    bool ipv6 = false;
```

Comment:
The `flutter run` command has support for ipv6 regardless of target.

Why does this change to `attach` make ipv6 only work for fuchsia?

Reply:
ipv6 was never set previously, so it defaulted to false.